From 57d034ce9f16c883f2f91dd2cc3b3070a8e665e2 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 18 Dec 2014 23:19:19 -0500
Subject: [PATCH 1/2] Add an XSetting for the session bus ID

This will allow us to identify situations where we are not on
the same session bus as the xsettings provider.

https://bugzilla.gnome.org/show_bug.cgi?id=671802
---
 gdk/x11/gdksettings.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gdk/x11/gdksettings.c b/gdk/x11/gdksettings.c
index 62a4ea3..620a58e 100644
--- a/gdk/x11/gdksettings.c
+++ b/gdk/x11/gdksettings.c
@@ -58,6 +58,7 @@ static const struct {
   {"Gtk/ShellShowsAppMenu",   "gtk-shell-shows-app-menu"},
   {"Gtk/ShellShowsMenubar",   "gtk-shell-shows-menubar"},
   {"Gtk/ShellShowsDesktop",   "gtk-shell-shows-desktop"},
+  {"Gtk/SessionBusId",        "gtk-session-bus-id"},
   {"Gtk/DecorationLayout",    "gtk-decoration-layout"},
   {"Gtk/TitlebarDoubleClick", "gtk-titlebar-double-click"},
   {"Gtk/TitlebarMiddleClick", "gtk-titlebar-middle-click"},
-- 
2.3.6

